misk/src/test/java/misk/web/extractors/JavaPathParamDispatchTest.java

```diff
@@ -0,0 +1,73 @@
+package misk.web.extractors;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.Module;
+import java.io.IOException;
+import javax.inject.Inject;
+import misk.MiskModule;
+import misk.testing.MiskTest;
+import misk.testing.MiskTestModule;
+import misk.testing.TestWebModule;
+import misk.web.Get;
+import misk.web.PathParam;
+import misk.web.ResponseContentType;
+import misk.web.WebActionModule;
+import misk.web.WebModule;
+import misk.web.actions.WebAction;
+import misk.web.jetty.JettyService;
+import misk.web.mediatype.MediaTypes;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@MiskTest(startService = true)
+public class JavaPathParamDispatchTest {
+  @MiskTestModule final Module module = new TestModule();
+
+  public enum ObjectType {
```

Comment:
ResourceType ?

ObjectType sounds like java.lang.Object which was threw me off for a sec